### DISCUSSION TITLE
Error! Binaries missing / corrupted

### DISCUSSION THREAD

**Discussion by blockx** [jely2002/youtube-dl-gui#485] [General]
Hi,
I'm trying to download the audio in the playl;ist at https://www.youtube.com/playlist?list=OLAK5uy_n0FNHGRkl7vpD1YfOVXMTouiHWgri1cbw but I get an error as follows:

`Error! Binaries missing/corrupted.
Please restart the app and disable antivirus`

I am not using any antivirus except that which is contained in Windows so can anyone tell me what is going on please ?

**Comment by AironT** [comment]
Confirming this issue. Windows defender only. Tried to disable it, not helpful. 
Have tried many random videos, and none of them worked.
Maybe some sort of youtube changed protocols. Not sure, just thoughts.

![image](https://github.com/jely2002/youtube-dl-gui/assets/6660183/3fc84337-3c85-486a-af44-7df2036ce49b)

**Reply by scorbun** [reply]
Hey so basically you've installed an old version that no longer works as explained [here](https://github.com/jely2002/youtube-dl-gui/issues/494), if you simply uninstall whatever you have currently & instead get your files from over [here](https://github.com/StefanLobbenmeier/youtube-dl-gui/releases/tag/v2.4.10), that should solve any issues, hope this helps

**Reply by blockx** [reply]
If you had actually tried this yourself then you would know that what you say is not true and the problem exists even with the most recent version of the downloader.

**Reply by amielitos** [reply]
Is this issue now resolved? Try downloading only the "exe" file and not the setup. Worked to me.

**Comment by amielitos** [comment]
What? this is a Github discussion page not Reddit. No need to use curse
words.

My fix worked on mine that's why I shared it. If it didn't work for you
that's not my problem.

**Comment by jely2002** [comment]
Hey all, thanks for the help provided. This has since been fixed in v3 of this app, closing this discussion now as resolved.
